db_test.go

```diff
@@ -226,6 +225,78 @@ func (s *SurrealDBTestSuite) TestDelete() {
 	s.Require().NoError(err)
 }
 
+func (s *SurrealDBTestSuite) TestFetch() {
+	// Define initial user slice
+	userSlice := []testUser{
+		{
+			ID:       "users:arthur",
+			Username: "arthur",
+			Password: "deer",
+			Friends:  []string{"users:john"},
+		},
+		{
+			ID:       "users:john",
+			Username: "john",
+			Password: "wolf",
+			Friends:  []string{"users:arthur"},
+		},
+	}
+
+	// Initialize data using users
+	for _, v := range userSlice {
+		data, err := s.db.Create(v.ID, v)
+		s.NoError(err)
+		s.NotNil(data)
+	}
+
+	// User rows are individually fetched
+	s.Run("Run fetch for individual users", func() {
+		for _, v := range userSlice {
+			res, err := s.db.Query("select * from $table fetch $fetchstr;", map[string]interface{}{
+				"record":   v.ID,
+				"fetchstr": "friends.*",
+			})
+			// TODO: This should be fixed once the code is fixed
+			s.Error(err)
```

Comment:
Could you please change this from `Error` to `NoError`, use `s.Skip` at the start of the test, and create a ticket that this needs to be resolved please? 🙏
```suggestion
			s.NoError(err)
```